### PR TITLE
TM-3732 update/employee agenda search location

### PR DESCRIPTION
**Pull request by andrews-john**
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2430)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/331)

### CODE REVIEW

File: talentmap_api/fsbid/services/agenda_employees.py [hunk 1]
```diff
@@ -223,6 +223,7 @@ def fsbid_agenda_employee_to_talentmap_agenda_employee(data, cdos=[]):
     if pydash.ends_with(fullName, "Nmn"):
         fullName = fullName.rstrip(" Nmn")
     cdo = None
+
```
Comment (SidelineCory24) [inline talentmap_api/fsbid/services/agenda_employees.py:226]: can remove

**Comment by SidelineCory24** [comment]
need to verify correct data is coming from WS, see [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/331)

**Comment by SidelineCory24** [comment]
This PR and the [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/331) are blocked by a WS issue.

**Comment by scott062** [comment]
This should now be unblocked - see nested payload in dev1 for updated payloads

**Pull request closed without merging**